debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -257,7 +257,22 @@ private Long getEndOffsetOfDbHistoryTopic(Long previousEndOffset, KafkaConsumer<
 
         return endOffset;
     }
-
+    
+    @Override
+    public boolean exists() {
+        boolean exists = false;
+        try (KafkaConsumer<String, String> historyConsumer = new KafkaConsumer<>(consumerConfig.asProperties());) {                        
```

Comment:
Just a question - will this work even if the topic is missing? I think we need to guard against both cases. Sorry if i was not clear.

Reply:
Yes, I tested both cases